index.html

```diff
@@ -104,9 +104,35 @@ <h2 class="work-title">Tonic</h2>
                         <li>CSS</li>
                         <li>JavaScript</li>
                     </ul>
-                    <a href="#" class="btn">See Project</a>
+                    <a class="btn" onclick="openPopup()">See Project</a>
```

Comment:
- [ ] All your `see project` buttons are supposed to open up a pop up window. In your implementation, This is the only button that pops up a window when clicked. The rest navigate back to the top of the page. Please make sure all `see project buttons` can pop up a details window of the project they appear under 👍🏾